.evergreen/run-load-balancer.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
This script comes from https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/run-load-balancer.sh

Reply:
Done.